assets/textures/gui/bevel/plusButtonDisabled.png.import

```diff
@@ -0,0 +1,34 @@
+[remap]
+
+importer="texture"
+type="StreamTexture"
+path="res://.import/plusButtonDisabled.png-d1d44538e5c6da836455b091bb723857.stex"
+metadata={
+"vram_texture": false
+}
+
+[deps]
+
+source_file="res://assets/textures/gui/bevel/plusButtonDisabled.png"
+dest_files=[ "res://.import/plusButtonDisabled.png-d1d44538e5c6da836455b091bb723857.stex" ]
+
+[params]
+
+compress/mode=0
+compress/lossy_quality=0.7
+compress/hdr_mode=0
+compress/bptc_ldr=0
+compress/normal_map=0
+flags/repeat=0
+flags/filter=true
+flags/mipmaps=false
```

Comment:
Mipmaps should be enabled to avoid potential quality loss, most other gui textures have this on.